src/main/java/stepDefs/uiSteps/HomeSteps.java

```diff
@@ -1,10 +1,37 @@
 package stepDefs.uiSteps;
 
 import core.TestContext;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
 
 public class HomeSteps {
 
-    private final TestContext testContext;
+    private TestContext testContext;
 
     public HomeSteps(TestContext testContext) {this.testContext = testContext;}
+    @Given("I login to salesforce app")
```

Comment:
Looks like "login to salesforce app" step already exists in LoginSteps class. Can you please check and remove the duplicate?

Reply:
fixed